app/src/userpages/components/StreamPage/New.jsx

```diff
@@ -173,18 +198,48 @@ const UnstyledNew = (props) => {
     const contentChangedRef = useRef(false)
     const dispatch = useDispatch()
     const { api: confirmExitDialog } = useModal('confirmExit')
-
+    const { load: loadIntegrationKeys, ethereumIdentities } = useEthereumIdentities()
+    const [domains, setDomains] = useState([])
+    const [loadingDomains, setLoadingDomains] = useState(true)
+    const ethereumIdentitiesRef = useRef([])
+    ethereumIdentitiesRef.current = ethereumIdentities
     const isMounted = useIsMounted()
 
+    const loadDomains = useCallback(async () => {
+        const addresses = ethereumIdentitiesRef.current.map(({ json }) => json.address)
```

Comment:
If you replace the addresses here with some that own domains, eg.

```js
// const addresses = ethereumIdentitiesRef.current.map(({ json }) => json.address)
const addresses = ['0xa729addefe1fa7bce87053ed55d55edddd13de60']
```

You can see some actual results.